<ui version="4.0" >
 <class>CommitMessage</class>
 <widget class="QWidget" name="CommitMessage" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>659</width>
    <height>622</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Commitmessage</string>
  </property>
  <layout class="QVBoxLayout" >
   <item>
    <widget class="QSplitter" name="m_MainSplitter" >
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <widget class="QFrame" name="m_ReviewFrame" >
      <property name="frameShape" >
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow" >
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout" >
       <item>
        <widget class="QLabel" name="m_Reviewlabel" >
         <property name="text" >
          <string>Review affected items</string>
         </property>
         <property name="alignment" >
          <set>Qt::AlignCenter</set>
         </property>
         <property name="wordWrap" >
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTreeView" name="m_CommitItemTree" >
         <property name="verticalScrollMode" >
          <enum>QAbstractItemView::ScrollPerPixel</enum>
         </property>
         <property name="rootIsDecorated" >
          <bool>false</bool>
         </property>
         <property name="itemsExpandable" >
          <bool>false</bool>
         </property>
         <property name="sortingEnabled" >
          <bool>true</bool>
         </property>
         <property name="allColumnsShowFocus" >
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" >
         <item>
          <widget class="QPushButton" name="m_HideNewItems" >
           <property name="text" >
            <string>Hide new items</string>
           </property>
           <property name="checkable" >
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="m_MarkUnversioned" >
           <property name="toolTip" >
            <string>Mark all new e.g. not versioned items for add and commit.</string>
           </property>
           <property name="text" >
            <string>Select new items</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="m_UnmarkUnversioned" >
           <property name="toolTip" >
            <string>Unmark all unversioned items so they will be ignored.</string>
           </property>
           <property name="text" >
            <string>Unselect new items</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="m_DiffItem" >
           <property name="toolTip" >
            <string>Generates and display difference against repository of selected item</string>
           </property>
           <property name="text" >
            <string>Diff highlighted item</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
      <zorder>m_HeadLabel</zorder>
      <zorder>m_LogEdit</zorder>
      <zorder>frame</zorder>
      <zorder>frame</zorder>
      <zorder>m_CommitItemTree</zorder>
      <zorder>m_Reviewlabel</zorder>
      <zorder></zorder>
     </widget>
     <widget class="QFrame" name="m_EditFrame" >
      <property name="frameShape" >
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow" >
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2" >
       <item>
        <widget class="QLabel" name="m_HeadLabel" >
         <property name="text" >
          <string>Enter a log message</string>
         </property>
         <property name="alignment" >
          <set>Qt::AlignCenter</set>
         </property>
         <property name="wordWrap" >
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="KTextEdit" name="m_LogEdit" />
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QGridLayout" >
     <item row="1" column="0" >
      <widget class="KComboBox" name="m_LogHistory" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Fixed" hsizetype="Minimum" >
         <horstretch>2</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="toolTip" >
        <string>Last used log messages</string>
       </property>
       <property name="duplicatesEnabled" >
        <bool>false</bool>
       </property>
       <item>
        <property name="text" >
         <string/>
        </property>
       </item>
      </widget>
     </item>
     <item row="0" column="0" >
      <widget class="QLabel" name="m_LogLabel" >
       <property name="text" >
        <string>Or insert one of the last:</string>
       </property>
       <property name="wordWrap" >
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="1" >
      <widget class="QPushButton" name="m_insert_file_button" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Fixed" hsizetype="Minimum" >
         <horstretch>1</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text" >
        <string>Insert Textfile</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <item>
      <widget class="DepthSelector" native="1" name="m_DepthSelector" />
     </item>
     <item>
      <widget class="QCheckBox" name="m_keepLocksButton" >
       <property name="toolTip" >
        <string>If checked commit will not release locks.</string>
       </property>
       <property name="text" >
        <string>Keep locks</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
  <zorder>m_MainSplitter</zorder>
  <zorder>m_ReviewFrame</zorder>
  <zorder>frame</zorder>
 </widget>
 <layoutdefault spacing="2" margin="2" />
 <customwidgets>
  <customwidget>
   <class>KComboBox</class>
   <extends>QComboBox</extends>
   <header>kcombobox.h</header>
  </customwidget>
  <customwidget>
   <class>KTextEdit</class>
   <extends>QTextEdit</extends>
   <header>ktextedit.h</header>
  </customwidget>
  <customwidget>
   <class>DepthSelector</class>
   <extends>QWidget</extends>
   <header>src/ksvnwidgets/depthselector.h</header>
  </customwidget>
 </customwidgets>
 <includes>
  <include location="local" >ktextedit.h</include>
  <include location="local" >kcombobox.h</include>
  <include location="local" >src/ksvnwidgets/depthselector.h</include>
 </includes>
 <resources/>
 <connections>
  <connection>
   <sender>m_LogHistory</sender>
   <signal>activated(int)</signal>
   <receiver>CommitMessage</receiver>
   <slot>slotHistoryActivated(int)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel" >
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_MarkUnversioned</sender>
   <signal>clicked()</signal>
   <receiver>CommitMessage</receiver>
   <slot>slotMarkUnversioned()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel" >
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_UnmarkUnversioned</sender>
   <signal>clicked()</signal>
   <receiver>CommitMessage</receiver>
   <slot>slotUnmarkUnversioned()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel" >
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_DiffItem</sender>
   <signal>clicked()</signal>
   <receiver>CommitMessage</receiver>
   <slot>slotDiffSelected()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel" >
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_HideNewItems</sender>
   <signal>toggled(bool)</signal>
   <receiver>CommitMessage</receiver>
   <slot>hideNewItems(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel" >
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_insert_file_button</sender>
   <signal>clicked()</signal>
   <receiver>CommitMessage</receiver>
   <slot>insertFile()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel" >
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
